### DISCUSSION TITLE
AI也能踢足球!?

### DISCUSSION THREAD

**Discussion by boson13579** [1101-GS4719-C/Python-Community-GS4719-C#1447] [General]
突然想到，如果有可以下棋的AI，那會不會有可以參與其他競技項目的AI呢?抱著這樣的想法，在網路搜尋了一陣子之後，找到了這篇文章。文章中說到，有個比賽叫做「機器人足球世界杯」，參與其中的科學家目的之一就是要讓機器人能在2050年時和世界盃冠軍隊一較高下。但文章中也有說，這個目標以現今的技術來說，還有很大的進步空間，因為其實足球不只需要運動能力，也需要智能、判斷力和默契，而這些問題不知何時才能解決。
文章連結：https://www.bbc.com/zhongwen/trad/science-58885123

**Comment by quan0715** [comment]
我相信對賽局的判斷是很好解決的。如同下棋一樣，AI可以很好的模仿一個賽局的進行，但是回到問題本質，大家會想看機器踢出來的足球嗎？世界杯會紅遍全球是因為大家對足球的熱愛，每一位球員不同的球風、華麗美技，群眾，這些都是令比賽緊張刺激的因素，先不說硬體設施能否跟上AI運算，人們會享受在一場只有算法而無激情的比賽嗎？

**Comment by brian431** [comment]
讓ai進行運動項目感覺是很可以的。不過技術瓶頸應該反而會出現在機器人的部分而不是ai 的部分。且我認為讓機器人踢足球之類的感覺沒什麼可看性？這種東西還是看人類比較有感覺ㄅ。

**Comment by aa103967622** [comment]
如果你想看看ai 踢足球的未來發展長什麼樣子，我建議你可以去看《藍色監獄》這部作品。這部作品是在講述說，一群有潛力高中生參加最強攻擊手的選拔，並在一個叫做藍色監獄的訓練營裡面閉關修煉。其中ai 輔助訓練和比賽是一大特色，比如說，他們在練習射門時，會有設定守門員，即時針對你的球路進行判斷及攔截，把你的射門球路訓練得愈加刁鑽，從而實力突飛猛進，甚至，也能跟ai 球員進行練習賽！
